### DISCUSSION TITLE
How to properly install packages

### DISCUSSION THREAD

**Discussion by Ghostriderboi** [ppkantorski/Ultrahand-Overlay#218] [Q&A]
https://switchway.net/set-up-game-cheats/
For context, btw im following this guide^ for using cheats. im new to the switch cfw scene im coming from xbox RGH modding. Anyway,
 i just got ultrahand and it says "Place your custom package.ini package file in your Ultrahand package directory (/switch/.packages/<PACKAGE_NAME>/). " BUT when i go to one of the ultrahand packages like "Mod alchemist" repo it says to put the whole mod alchemist folder in /switch/.packages/.
so do i put the packages.ini from mod alchemist or the whole folder in .packages? also is the "See [Ultrahand Packages](https://github.com/ppkantorski/Ultrahand-Packages) for a comprehensive list of known packages." hyperlink the only existing packages we can use for ultrahand as of now? 
thanks
![mod alch](https://github.com/user-attachments/assets/a3d9c894-3ae4-46c3-8738-f0150d6fa191)

**Comment by ppkantorski** [comment]
custom packages go in their own folder within `/switch/.packages/`.  root packages goes in `/switch/.packages/`.  the difference is that root packages don't have their own page, and are drawn on the packages menu itself while packages within their own folder will show up under "Packages".

in this case it would be ```/switch/.packages/Mod Alchemist/package.ini```
